platform/rct/Makefile

```diff
@@ -16,7 +16,7 @@ OBJCOPY := $(CROSS_COMPILE)objcopy
 ASL_FW_POSTPROCESS := asl_f4_fw_postprocess
 
 #specify the use of the external crystal
-BOARD_DEFINES = -DPLL_SOURCE_HSE -DUSE_I2C1
+BOARD_DEFINES = -DPLL_SOURCE_HSE -DUSE_I2C1 -fno-zero-initialized-in-bss
```

Comment:
I think you're misunderstanding the flag. This will help: http://stackoverflow.com/questions/8721475/if-a-global-variable-is-initialized-to-0-will-it-go-to-bss